lib/school.rb

```diff
@@ -0,0 +1,31 @@
+require 'time'
+
+class School
+ attr_reader :hours_in_school_day,
+             :student_names,
+             :start_time
+
+  def initialize(hours_in_school_day, start_time)
+    @hours_in_school_day = hours_in_school_day
+    @start_time = start_time
+    @student_names = []
+    @is_full_time = true
+  end
+
+  def add_student_name(x)
+    @student_names << x
+  end
+
+  def end_time
+ time = (hours_in_school_day.to_i + start_time.to_i)
+ # Time.strptime(time, "%I:%M %P").strftime("%H:%M")
+  end
+
+   def is_full_time?
+     @is_full_time
```

Comment:
This only ever reports true, and does not check if it is not a full time school.